### PR TITLE
chore: update how to contribute

### LINKED ISSUE #269: Update CONTRIBUTING
Stuff in CONTRIBUTING.md is out of date.

Spec is using ReSpec again, and should suggest using tidy when possible.

### LINKED ISSUE #273: Remove (or update?) Makefile
Seems pointless now, https://github.com/w3c/permissions/blob/main/Makefile

@marcoscaceres is there any ReSpec-y thing it should be doing, or do we just nuke it from orbit?

### PR DESCRIPTION
**Pull request by marcoscaceres**
closes #273 
closes #269

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  - When relevant, include `closes #` and the issue number.
+  - The commit message should be short and concise.
+  - Prefix the commit message with one of the following:
+    - `Editorial:` -for editorial/non-normative change to the specification.
+    - `Chore:` - when fixing ReSpec stuff or other non-spec related stuff (e.g., CI).
+    - Normative changes should not be prefixed with anything. These go into the changelog.
+
+## Running HTML5 Tidy
+
+Please make sure you have [HTML5 Tidy](https://github.com/w3c/tidy-html5) installed.
```
Comment (miketaylr) [inline .github/CONTRIBUTING.md:33]: nit: that URI redirects to https://github.com/htacg/tidy-html5, not sure if it's worth updating or if you prefer living w/ the redirect

Comment (miketaylr) [reply .github/CONTRIBUTING.md:33]: Thinking with my "new contributor hat" and trying to figure out how to download and install tidy, the README in this repo isn't helpful. Building from source feels like overkill. Going to http://www.html-tidy.org/... it's a non-secure site, so I totally don't want to install the binaries there.

(edit: they _do_ have a secure version of the binaries page, at https://binaries.html-tidy.org/)

Grabbing the latest release from https://github.com/htacg/tidy-html5/releases is probably fine, or the site has links to package managers, e.g., `brew install tidy-html5` for macOS.

Perhaps we could add a sentence after `Please make sure you have [HTML5 Tidy](https://github.com/w3c/tidy-html5) installed.` like:

> Please make sure you have [HTML5 Tidy](https://github.com/w3c/tidy-html5) installed. You can download the [latest stable release from GitHub](https://github.com/htacg/tidy-html5/releases) or follow the "Get Tidy" instructions at http://www.html-tidy.org/.

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+Don't use the the one that ships with \*nix systems.
+
+You can confirm check the version by running:
+
+```bash
+tidy --version  #HTML Tidy for HTML5 (experimental) for ...
```
Comment (miketaylr) [inline .github/CONTRIBUTING.md:39]: Mine say `HTML Tidy for Apple macOS version 5.8.0`. Is there a minimum supported version folks should be aiming for?

(the "old" one reports `HTML Tidy for Mac OS X released on 31 October 2006 - Apple Inc. build 1045`

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (marcoscaceres) [reply .github/CONTRIBUTING.md:33]: Yeah, brew instructions will do. Alternative, we just remove mention of it. GitHub actions handles it for random contributors.

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (marcoscaceres) [reply .github/CONTRIBUTING.md:39]: I’ll need to check what the latest is. But generally latest is what people should be on. It’s only gets an update like once a year.

**Review by miketaylr — APPROVED** [review]
This is a big improvement right now - we can make the instructions more useful after it lands.

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (miketaylr) [reply .github/CONTRIBUTING.md:39]: Got it - the `#HTML Tidy for HTML5 (experimental) for ...` comment made me think my version wasn't correct. But if we just write "the latest version" it'll be fine 99% of the time.

**Pull request merged**